doc/tutorial.rst

```diff
@@ -586,8 +573,9 @@ The `static/js/src/simplevideo_edit.js` JavaScript file should look like this: :
           maxwidth: $(element).find('input[name=maxwidth]').val(),
           maxheight: $(element).find('input[name=maxheight]').val()
         };
+        runtime.notify('save', {state: 'start'});
         $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
-          window.location.reload(false);
+          runtime.notify('save', {state: 'end'});
```

Comment:
@Kelketek I considered that, but the `studio_view` doesn't work with current `SimpleVideoBlock`, because `<vertical_demo>` doesn't implement a studio view. In the next version of the tutorial, I'd really like to remove `vertical_demo` and `thumbs` from the workbench scenarios and only put `<simplevideo href="https://vimeo.com/46100581" maxwidth="800" />` in there. I'd put the children related stuff to a dedicated section.